aggregates/src/androidMain/kotlin/org/dhis2/mobile/aggregates/data/DataSetInstanceRepositoryImpl.kt

```diff
@@ -676,8 +680,17 @@ internal class DataSetInstanceRepositoryImpl(
         }
             ?: if (dataElementValueType is InputType.MultiText) InputType.MultiText else InputType.OptionSet
 
+        val categoryOptionComboSuffix =
+            if (isDefaultCategoryCombo) {
+                ""
+            } else {
+                " / ${categoryOptionCombo?.displayName()}"
+            }
+
+        val dataElementLabel = "${dataElement.displayFormName()}$categoryOptionComboSuffix"
```

Comment:
maybe all the getting the label could be extracted to a function

Reply:
Makes sense, I'll update it